src/System.Text.Primitives/System/Text/Encoding/Internal/ParsingTrie.cs

```diff
@@ -81,12 +81,16 @@ public struct Node
         private struct Suffix : IComparable<Suffix>
         {
             public int SymbolIndex;
-            public ReadOnlySpan<byte> Bytes;
+            public byte[] Bytes;
 
             public Suffix(int symbolIndex, ReadOnlySpan<byte> bytes)
             {
                 SymbolIndex = symbolIndex;
-                Bytes = bytes;
+
+                // HACKHACK: Keeping Bytes as a Span property on Suffix will cause crashing in .NET Core 2.0.
+                //           Storing as pure array for now until we can re-visit.
```

Comment:
@VSadov Is this a scenario that Roslyn should be guarded against?  @shiftylogic could you create a github issue tracking this and reference it from the comment above?